pgmpy/sampling/base.py

```diff
@@ -418,3 +419,35 @@ def _return_samples(return_type, samples):
             return samples
     else:
         return samples
+
+
+def _map_to_state_name(model, samples):
+    """
+        A utility function to map samples to corresponding state name
```

Comment:
Could you add an example here as well ? It's difficult to understand what exactly it's supposed to do by looking at the code.

Reply:
@ankurankan It is an internal method and is not supposed to be used by user. That's why I didn't added it in the first place

Reply:
But it would be good to have an example for our own reference later.